charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -90,13 +90,13 @@ spec:
         {{ if or .ci (not (hasKey . "ci")) -}}
         initContainers:
           - name: install
-            image: jnewland/git-and-stuff
+            image: jnewland/git-and-stuff:v0.1.2
             imagePullPolicy: Always
-            securityContext:
-              runAsUser: {{ (.containerSecurityContext).runAsUser | default 1001 }}
+            # securityContext:
```

Comment:
Why it is commented?

Reply:
because this container doesn't have user ID 1001, and git commands are not working